phoenix-core/src/main/java/org/apache/phoenix/schema/PTableImpl.java

```diff
@@ -230,6 +254,7 @@
         private Boolean useStatsForParallelization;
         private long viewTTL;
         private long viewTTLHighWaterMark;
+        private Map<String, String> values = new HashMap<>();
```

Comment:
"values" seem like a bad name for maps that has both keys and values - how about "properties"? Looks like that's what are being put there.